docs/BR.md

```diff
@@ -1099,8 +1103,40 @@ No stipulation.
 
 ### 4.3.1 CA actions during certificate issuance
 
+#### 4.3.1.1 Manual authorization of certificate issuance for Root CAs
+
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Precertificate [RFC 6962] is issued, effective 2024-09-15 the CA SHOULD implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements. Effective 2025-03-15, the CA SHALL implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements.
+
+Methods used to produce a certificate containing the to-be-signed Certificate content include, but are not limited to:
+
+1. Sign the `tbsCertificate` with a "dummy" Private Key whose Public Key component is not certified by a Certificate that chains to a publicly-trusted CA Certificate; or
+2. Specify a static value for the `signature` field of the Certificate ASN.1 SEQUENCE.
+
+CAs MAY implement their own certificate Linting tools, but CAs SHOULD use the Linting tools that have been widely adopted by the industry (see https://cabforum.org/resources/tools/). 
+
+CAs are encouraged to contribute to open-source linting projects, such as by:
+
+- creating new or improving existing lints,
+- reporting Requirements that are not covered by existing lints ("unlinted issues"), 
```

Comment:
```suggestion
- reporting requirements that are not covered by existing lints ("unlinted issues"), 
```

Reply:
The Defined Term "Requirements" refers to the current document as a whole.

Reply:
I'd rather use a different word to make it more clear to readers. Please let me know what you think about the following:

```suggestion
- reporting issues of technical checks that are not covered by existing lints ("unlinted issues"), 
```

Reply:
I'm having some trouble understanding what "issue" is in this context. I think it's shorthand for "Github issue" or similar, so perhaps this is more direct:

```
- notifying maintainers of Linting software of checks that are not covered by existing lints, 
```